viewer/packages/camera-manager/tests/cameraControls.test.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import * as THREE from 'three';
+import 'jest-extended';
+import ComboControls from '../src/ComboControls';
+
+describe('Camera Controls', () => {
```

Comment:
I think the file should be renamed to ComboControls.test.ts and this change to
```suggestion
describe('ComboControls', () => {
```

Also - we've moved test to be inlined in src/ - can this be done here as well?

Reply:
I personally don't really like / see the reason to have tests inlined in /src since we are now in a much more confined scope. I felt that tests where in /src simply because it was impossible to find the test for a given file with the previous directory structure. Also conceptually it is strange to have tests in /src since... well... it isn't source code...

Reply:
My main argument for having them side by side is that it becomes hard to move classes around when tests are in separate folders. I always felt this was a PITA in .NET where tests quickly would end up in mismatching folders or with mismatching filenames...

Reply:
I see that this is a pain, but I still think that this is an effect of having a too deep / complex folder structure and that this will be much less of a problem going forward. If this assumption turns out to be false I will bite the bullet and inline all the tests.

Reply:
The file and test description should _at least_ be renamed to match the file being tested

Reply:
Agreed, will fix asap

Reply:
Wait, this should already been done????